core/components/mediamanager/model/mediamanager/classes/files.class.php

```diff
@@ -1579,6 +1578,7 @@ public function saveFile($fileId, $rawData)
         if ($createFileVersion) {
             $this->saveFileVersion($file->get('id'), $data, $actionName);
         }
+        $file->save();
```

Comment:
We moved the save at the end, because on some edge cases it was possible to get the version increased, but the file version creation failed, resulting in inconsistent data